src/main/java/io/confluent/examples/streams/interactivequeries/kafkamusic/KafkaMusicExample.java

```diff
@@ -290,14 +295,15 @@ static KafkaStreams createChartsStreams(final String bootstrapServers,
     final StreamsBuilder builder = new StreamsBuilder();
 
     // get a stream of play events
-    final KStream<String, PlayEvent> playEvents = builder.stream(Serdes.String(),
-        playEventSerde,
-        PLAY_EVENTS);
+    final KStream<String, PlayEvent> playEvents = builder.stream(PLAY_EVENTS, Consumed.with(Serdes.String(),
+                                                                                            playEventSerde));
 
     // get table and create a state store to hold all the songs in the store
     final KTable<Long, Song>
-        songTable =
-        builder.table(Serdes.Long(), valueSongSerde, SONG_FEED, ALL_SONGS);
+            songTable =
```

Comment:
indention

Can we add a style check and fail the build on wrong indention \cc @miguno